src/index.ts

```diff
@@ -103,9 +104,26 @@ function processExtensions(definition: Definition): Definition {
             assert.fail(`'${definition.model}' has function exposes which is not allowed`);
         }
 
+        const actionExposes: Expose[] = [];
+
+        if (exposes) {
+            const filteredExposes: Expose[] = [];
```

Comment:
Instead of doing the filtering both here and in `if (ext.exposes) {`. Can we do it just one time above `if (actionExposes.length > 0) {`?

Reply:
Done, not sure why I did this in the first place.